src/commands/main.js

```diff
@@ -1,13 +1,13 @@
 const Discord = require('discord.js')
-const dayjs = require('dayjs')
-const relativeTime = require('dayjs/plugin/relativeTime')
-dayjs.extend(relativeTime)
 
 const { system } = require('../utils')
 const version = require('../../package.json').version
 
 module.exports = async function (message, parent) {
-  let summary = `Dokdo v${version}, discord.js \`${Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at ${dayjs(system.processReadyAt()).fromNow()}, bot was ready at ${dayjs(parent.client.readyAt).fromNow()}.\n`
+  const makeRelativeTime = (time) => `<t:${time}:R>`
```

Comment:
This would be better if you separate it into a util file.

Also, it would be better to pass date as an argument
```suggestion
  const makeRelativeTime = (date) => `<t:${Math.floor(d/1000)}:R>`
```